make/photon/clair-adapter/Dockerfile.binary

```diff
@@ -3,5 +3,5 @@ FROM golang:1.13.4
 ADD .   /go/src/github.com/goharbor/harbor-scanner-clair/
 WORKDIR /go/src/github.com/goharbor/harbor-scanner-clair/
 
-RUN export GOOS=linux GO111MODULE=on CGO_ENABLED=0 && \
-	go build github.com/goharbor/harbor-scanner-clair/cmd/harbor-scanner-clair
+RUN export GOFLAGS=-mod=vendor GOOS=linux GO111MODULE=on CGO_ENABLED=0 && \
+	go build -o harbor-scanner-clair cmd/harbor-scanner-clair/main.go
```

Comment:
why not use "go build -o harbor-scanner-clair ./cmd/harbor-scanner-clair" as other build scripts

Reply:
`go build -o harbor-scanner-clair cmd/harbor-scanner-clair/main.go` keep the same in Makefile of `harbor-scanner-clair`